core/src/blocktree/rocks.rs

```diff
@@ -18,8 +18,7 @@ use std::path::Path;
 
 // A good value for this is the number of cores on the machine
 const TOTAL_THREADS: i32 = 8;
-const MAX_WRITE_BUFFER_SIZE: u64 = 256 * 1024 * 1024; // 256MB
-const MIN_WRITE_BUFFER_SIZE: u64 = 64 * 1024; // 64KB
+const MAX_WRITE_BUFFER_SIZE: u64 = 512 * 1024 * 1024; // 256MB
 
```

Comment:
This is now 512.
Also this will cause OOMs.

Reply:
Why OOMs?